### ISSUE TITLE
TypeScript support

### ISSUE THREAD

**Issue by diegoaguilar** [saudslm/recharge-api-node#9]
I'm trying to use [recharge-api-node](https://github.com/saudslm/recharge-api-node) in a TypeScript project.

When I tried to use ES6 module style import, I noticed this is only written in JavaScript. Do you have plans for including TypeScript support?

I may be interested in helping at least with basic types definitions.

**Comment by saudslm** [comment]
@diegoaguilar 

Yes, I do plan to incorporate types definitions but it may take sometime. If you can help with basic types definitions, that would be great.
